### DISCUSSION TITLE
Welcome to PSDocs Discussions!

### DISCUSSION THREAD

**Discussion by ElinoraWhite** [microsoft/PSDocs#100] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by BrooksV** [comment]
Hi Bernie,

I happen across the PSDocs & PSDocs.Azure projects while looking for how best to create and maintain documentation related to Azure Policy.

I use a home grown Policy-as-Code approach which uses a folder and file structure in a git repo for managing Policy configuration data and a DevOps YAML Pipeline to deploy the changes to Azure.

I currently have a readme.md file in each policy folder but I cam looking for a more manageable way to:
- Organize and standardize content creation and output
- Repurpose the content to generate Friendly User Documentation and Detailed Technical Documentation for Support
- Generate output as PDF posted to a SharePoint 365 Online Document folder and possibly a Static Web Site integrated into a SharePoint 365 Online Site

Liked the idea about extracting the Metadata from the configuration files but I am not using ARM templates to maintain and deploy policy.

I do use separate json files for the Policy Rule, Parameters, and Policy Metadata but use PSD1 files for Policy Definition Name, Type, Mode, Scope, Description, and Assignments. Using PSD1 files import as HashTables making them ideal as Splats to the Azure Powershell Policy CmdLets.

One problem I have is most of my team members are not programmers and have no experience with DevOps so what ever solution is implemented to create and maintain documentation, needs to be simple enough to understand and manage.

The questions I have are:

1. What is the difference between PSDocs & PSDocs.Azure?
2. Should I be using one over the other?
3. Do I need to use both?
4. Any suggestions?

Regards, Brooks
